pypowsybl/sensitivity.py

```diff
@@ -145,35 +145,35 @@ class Parameters:  # pylint: disable=too-few-public-methods
     .. currentmodule:: pypowsybl.sensitivity
 
     Args:
-        load_flow_parameters: parameters that are common to loadflow and sensitivity analysis
+        loadflow_parameters: parameters that are common to loadflow and sensitivity analysis
```

Comment:
We need to be careful with such renamings in the python API:
it will break existing user code.

So if we really want to change it, we need to identify it as a breaking change in the release note.

I would prefer not to mix it with this PR which is only about adding a new feature, well separated from this part of the code.

Reply:
Should I then undo all the renaming I did that does not concern the validation parameters (c++ and python), or only the ones in python that change the way the user can call parameters ?

Reply:
We can keep the changes that are not part of the python API, yes :)